aten/src/ATen/native/cuda/ForeachReduceOp.cu

```diff
@@ -28,8 +28,8 @@ template <
     int res_arg_index = 0>
 struct LpNormFunctor {
   static_assert(
-      NormType == 1 || NormType == 2,
-      "foreach_norm supports only L1 and L2 norm");
+      NormType == 1 || NormType == 2 || NormType == 1234567890,
```

Comment:
I'd introduce an enum, e.g.,

```c++
enum class NormType {
  L1, L2, LInf
};
```

Reply:
Oh this is p nice did not realize this was possible